experiments/Molecule_Generation/helper_funcs.py

```diff
@@ -4,6 +4,15 @@
 
 
 def canonicalize_smiles(smiles: str) -> str:
```

Comment:
Duplicate of the smiles utils?

Reply:
@tbennun So I didn't want to remove all of those until @szaman19 has a chance to think about how future experiment should be constructed.

Reply:
OK let’s leave it here for now